### ISSUE TITLE
about training

### ISSUE THREAD

**Issue by TnoobT** [hywang2002/MV-VTON#13]
您好，我想请问一下训练viton数据集要多久，另外显卡显存占用多少

**Comment by hywang2002** [comment]
Hi, thank you for your interest. In my impression, the model training on a single A100 requires about 60G memory and about 12 hours. Please note that this may be  a little different from the actual situation.

**Comment by TnoobT** [comment]
> Hi, thank you for your interest. In my impression, the model training on a single A100 requires about 60G memory and about 12 hours. Please note that this may be a little different from the actual situation.

请问训练稳定吗，我用idm训练，loss不收敛，并且inpainting的效果也不好
![image](https://github.com/user-attachments/assets/584d7a96-3eb3-4b59-89dc-a69d2a71b9eb)

**Comment by TnoobT** [comment]
> Hi, thank you for your interest. In my impression, the model training on a single A100 requires about 60G memory and about 12 hours. Please note that this may be a little different from the actual situation.

大佬求回复

**Comment by hywang2002** [comment]
We used the Paint by Example pretrained model as the initialization of u-net during training with about 40 epochs. The hyperparameters were based on DCI-VTON and no additional hyperparameter adjustments were made. I suggest that you could try:
1. Use the pretrained model to initialize u-net weights, such as Paint by Example.
2. First train u-net as an inpainting method, then train u-net and controlnet together.

**Comment by hywang2002** [comment]
Hope this will help you.

**Comment by TnoobT** [comment]
> Hope this will help you.

Thank you very much for the author's reply. May I ask what is the loss of mv vton at the beginning of training. How much can the loss be reduced after training. I want to train with your code

**Comment by hywang2002** [comment]
> > Hope this will help you.
> 
> Thank you very much for the author's reply. May I ask what is the loss of mv vton at the beginning of training. How much can the loss be reduced after training. I want to train with your code

You can try the specific value of loss yourself due to the long time. During the training, we not only paid attention to the change of loss, but also the performance of the model on the validation dataset. We recommend that you try it as well.
